packages/pwa-buildpack/package.json

```
@@ -52,12 +49,15 @@
     "workbox-webpack-plugin": "~3.6.1",
     "write-file-webpack-plugin": "~4.2.0"
   },
+  "devDependencies": {
+    "babel-cli": "~6.26.0",
+    "rimraf": "~2.6.2"
+  },
   "peerDependencies": {
     "babel-core": "^6.26.0",
     "babel-loader": "^7"
   },
-  "devDependencies": {
-    "babel-cli": "~6.26.0",
-    "rimraf": "~2.6.2"
+  "engines": {
+    "node": ">=8.0.0"
```

This should probably be updated to match our root config, [`"10.14.1"`](https://github.com/magento-research/pwa-studio/blob/release/2.0/package.json#L12).

---

Hey, you're right! Not technically part of this PR, but we might as well seal in the change.

```suggestion
    "node": ">=10.14.1"
```